frontend-instructor/src/components/courseManagement/QuizForm.jsx

```
@@ -0,0 +1,543 @@
+  isEditing = false,
+  quizId = null
+}) => {
+  const { fetchData } = useApi();
+  const [showQuestionForm, setShowQuestionForm] = useState(false);
+  const [editingIndex, setEditingIndex] = useState(-1);
+  const [newOptions, setNewOptions] = useState(['', '', '', '']);
+  const [error, setError] = useState('');
+
+  const initialValues = {
+    questions:
+      quizData && quizData.questions
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)